Sources/Rendering/OpenGL/Actor/index.js

```diff
@@ -152,7 +152,10 @@ function vtkOpenGLActor(publicAPI, model) {
   // Renders myself
   publicAPI.translucentPass = (prepass, renderPass) => {
     if (prepass) {
-      model.context.depthMask(false);
+      model.context.depthMask(
+        model._openGLRenderer.getSelector() &&
```

Comment:
We may want a configurable threshold on opacity, that enables picking for translucent actors
For example, we don't want to pick when the opacity is 0
This threshold can be ignored here as we want to completely disable rendering of translucent actors if they don't reach the threshold (if we only disable depth mask, the actors will be rendered anyway), so the threshold test could be when doing the actual rendering

Reply:
@bruyeret Sorry for not getting back to you sooner. We don't need a configurable opacity threshold for picking simply because if the actor's opacity is 0, it will not be rendered (even for selection). If users desire to ignore certain actors from selection, that can be done via `PickableOff`. WDYT?

Reply:
Yes that's ok, it works 👍